content/_components/onedrive/index.md

```diff
@@ -6,152 +6,158 @@ description: This is the component for working with Microsoft OneDrive storage s
-| `OAUTH_CLIENT_SECRET`| true | Microsoft Graph Application OAuth2 Client Secret | Can be found in your application page on [https://portal.azure.com](https://portal.azure.com) |
-| `ATTACHMENT_MAX_SIZE`| false | For `elastic.io` attachments configuration. Maximal possible attachment size in bytes. By default set to 1000000 and according to platform limitations CAN'T be bigger than that. | Up to `1000000` bytes|
+### Get New and Updated Objects Polling
 
-> Please Note: From the platform version [20.51](/releases/20/51) we deprecated the
-> component `LOG_LEVEL` environment variable. Now you can control logging level per each step of the flow.
+Retrieve all the updated or created objects within a given time range.
 
-### Credentials
+#### Configuration Fields
 
-To create new credentials you need to authorize in Microsoft system using OAuth2 protocol.
+* **Drive Identity** - (dropdown, required): OneDrive instance to work with
+* **Folders path** - (multiselect dropdown, required): Select folders to follow
+* **Include subfolders** - (checkbox, optional): If checked, trigger will follow to each subfolder of selected folder.
+    >**Please Note:** this will increase the number of API calls
 
-## Triggers
+* **Enable File Attachments** - (checkbox, optional): If checked, file will be uploaded to local storage and link provided in response
+* **Time stamp field to poll on** - (dropdown, optional, default `Last Modified`): Select which date will be used to track files - `Last Modified` or `Created`
+* **Emit Behavior** - (dropdown, optional, default `Emit individually`): Defines the way result objects will be emitted, one of `Emit page` or `Emit individually`.
+* **Page Size** - (number, optional, defaults to 999, max 999): Indicates the size of pages to be fetched per request
+* **Start Time** - (string, optional): The timestamp to start polling from (inclusive) - using ISO 8601 Date time utc format - YYYY-MM-DDThh:mm:ssZ. Default value is the beginning of time (January 1, 1970 at 00:00).
+* **End Time** - (string, optional): The timestamp to stop polling (exclusive) - using ISO 8601 Date time utc format - YYYY-MM-DDThh:mm:ssZ. Default value is flow execution time.
 
-### Get New And Updated Files Polling
+#### Input Metadata
 
-Triggers to get all new and updated files since last polling. Polling is provided by `lastModifiedDateTime` file's property.
+There is no Input or Output metadata in this trigger.
 
-![Get New And Updated Files Polling](img/get-new-and-update.png)
+#### Output Metadata
 
-#### List of Expected Config fields
+Depends on `Enable File Attachments` and `Emit behavior` fields.
+ * If `Emit behavior` field is equal to `Emit page` - object with property `results` that contains array of files
+ * If `Emit behavior` field is equal to `Emit individually`, file information will fulfill whole message
+ * If `Enable File Attachments` checked, for each file there will be additional field - `attachmentUrl`
```

Comment:
* If `Enable File Attachments` is checked, for each file there will be an additional field - `attachmentUrl`